make/Makefile.container.mk

```diff
@@ -44,7 +44,14 @@ container-build-operator: .ensure-operator-sdk-exists
 	cd "${ROOTDIR}/operator" && "${OP_SDK}" build --image-builder ${DORP} --image-build-args "--pull" "${OPERATOR_QUAY_TAG}"
 
 ## container-build: Build Kiali and Kiali operator container images
-container-build: container-build-kiali container-build-operator
+container-build:
+# On x86_64 machine, build both kiali and operator images.
+ifeq ($(GOARCH),amd64)
+	make container-build-kiali container-build-operator
```

Comment:
Don't call "make" directly. At minimum use `$(MAKE)` but even doing that is going to have unintended consequences.

Since all you are doing is restricting the dependent targets, do something like:

```
ifeq ($(GOARCH),amd64)
container-build: container-build-kiali container-build-operator
else
container-build: container-build-kiali
endif
```

We do similar things in `make/Makefile.operator.mk` , target `kiali-create` - so see that for an example.

Reply:
Done. Thanks for this :))